Update README.md for the Zimlet guide

ykomiyam opened the pull request:
* Removed the description about the pre-compiled version of mytest.jar.  The description is not applicable right now; there is no mytest.jar in the specified directory, and people have to compile it themselves.
* The zimbra-zimlet-sideloader package should be installed under the root privileges
* Explicitly described the nodejs version.  As for now, I tested and found that minimumly node -v should be v14.21.1 (on Ubuntu)
* Added a description about some L10N modification, and put a sample L10N file in Japanese.

barrydegraaff reviewed (changes requested):
Please do not remove the lines 17-24
Please fix typo in sudo jyum

ykomiyam commented:
> Please do not remove the lines 17-24 
> Please fix typo in sudo jyum

Thanks @barrydegraaff for the review.  I fixed typo of "yum".  
As for the lines 17-24, I couldn't download the mytest.jar from the specified place:
```
$ wget https://github.com/Zimbra/zm-extension-guide/releases/download/0.0.2/mytest.jar
--2022-12-01 16:53:56--  https://github.com/Zimbra/zm-extension-guide/releases/download/0.0.2/mytest.jar
Resolving github.com... 140.82.114.4
Connecting to github.com|140.82.114.4|:443... connected.
OpenSSL: error:1407742E:SSL routines:SSL23_GET_SERVER_HELLO:tlsv1 alert protocol version
Unable to establish SSL connection.
``` 
That's why I removed this paragraph.  If we keep this instruction, we should make mytest.jar downloadable.  Besides, as the version 0.0.8 has been released, why don't we specify the latest version instead of 0.0.2?

barrydegraaff commented:
Please update the link to point to the latest version which is: https://github.com/Zimbra/zm-extension-guide/releases/download/0.0.8/mytest.jar
